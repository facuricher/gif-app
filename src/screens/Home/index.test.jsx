import { render, screen } from "@testing-library/react"
import { Home } from "."

describe('test in <Home />', () => {
  test('must match snapshot', () => {
    const { container } =  render( <Home />);
    expect( container ).toMatchSnapshot();
  });
});
