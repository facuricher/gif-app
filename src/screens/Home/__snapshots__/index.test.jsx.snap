// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test in <Home /> must match snapshot 1`] = `
<div>
  <h1>
    Find your favorite gifs
  </h1>
  <form
    aria-label="form"
  >
    <input
      placeholder="search gifs"
      type="text"
      value=""
    />
  </form>
  <h3>
    Rick and Morty
  </h3>
  <h2>
    Loading...
  </h2>
  <div
    class="card-grid"
  />
</div>
`;
